read-fonts/src/collections/int_set/bitset.rs

```diff
@@ -11,29 +26,29 @@ use std::ops::RangeInclusive;
 // log_2(PAGE_BITS)
 const PAGE_BITS_LOG_2: u32 = PAGE_BITS.ilog2();
 
-/// An ordered integer (u32) set.
+/// A fast, efficient, sparse, & ordered u32 set. Not invertible.
```

Comment:
I'm not sure 'not invertible' is that meaningful, really? If it is, I would prefer to say something like, "for a higher-level API that supports inversion, use [`IntSet`]".